### DISCUSSION TITLE
Compressed symbols and serving them via HTTP

### DISCUSSION THREAD

**Discussion by t-b** [elmirjagudin/symstore#31] [Q&A]
I'm currently using symstore with compression (`-z`). And just tried to retrieve one of the compressed pdb files using symchk.exe. And the problem I have is that symchk.exe is not even trying to download the `pd_` file:

```
SYMSRV:  HTTPGET: //index2.txt
SYMSRV:  HttpQueryInfo: 801901f4 - HTTP_STATUS_SERVER_ERROR
SYMSRV:  HTTPGET: /ITCXOP2-64.pdb/8DB6890FB625415E8723A03EDD0596751/ITCXOP2-64.pdb
SYMSRV:  HttpQueryInfo: 801901f4 - HTTP_STATUS_SERVER_ERROR
SYMSRV:  RESULT: 0x801901F4
```

So now I'm wondering if someone else is using symstore with compression and is serving the symbols via http?

**Comment by t-b** [comment]
Looks like not.

**Reply by AnyOldName3** [reply]
We do this for OpenMW's symbol server, and Visual Studio can access the compressed `.pd_` files just fine. If I had to guess, maybe it's because you're serving up a 500 status code for the `.pdb` instead of a 404, and `symchk.exe` is taking that as a sign that it shouldn't bother asking about the `.pd_`.

**Reply by t-b** [reply]
@AnyOldName3 That could be it! Thanks for replying in this old and closed thread. I'll give that a shot and report back.

**Reply by AnyOldName3** [reply]
It's good that I did because I ran into the same problem myself a couple of days later and quickly knew what to do.
